cob_obstacle_distance/scripts/obstacle_publisher_node.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2014 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_obstacle_distance
+ *
+ * \author
+ *   Author: Marco Bezzon, email: Marco.Bezzon@ipa.fraunhofer.de
+ *
+ * \date Date of creation: July, 2015
+ *
+ * \brief
+ *   Simple Python node to publish obstacles to cob_obstacle_distance node.
+ *
+"""
+import time
+import roslib 
+import rospy
+
+from geometry_msgs.msg import Pose
+from moveit_msgs.msg import CollisionObject
+from shape_msgs.msg import SolidPrimitive
+from shape_msgs.msg import Mesh
+
+
+if __name__=="__main__":
+    rospy.init_node("simple_obstacle_pub")
+    root_frame = "/odom_combined"
```

Comment:
Test-script just for test. 
If it is desired to check the error behavior it is desired to enter another frame than that one from parameter server. Else there is no chance to check for errors. 

If it is desired to have a spawner node then this has to be written separate from a test script.